### ISSUE TITLE
replace "testable" by "suitable"

### ISSUE THREAD

**Issue by gernotstarke** [arc42/quality.arc42.org-site#90]
@Hruschka suggested to replace "testable" by "suitable", is it is more high-level and would cover functional aspects better.

what do you think, @ichsteffen ?

>
>Als Qualität verstehe ich den ja, aber wenn man aus den Qualitäten Requirements macht, dann macht „testbar“ keinen Sinn, denn >ALLE Requirements sollten testbar (überprüfbar) seinl
>Statt „testable“ wäre für mich „suitable“ der bessere Einstiegspunkt auf der Top-Ebene. Dann hätten wir für alle „functional-xxx“ >einen besseren Einstiegepunkt als „usable“, einen der die Business-Eignung eines Systems in den Vordergrund stellt, und nicht >die Benutzerfreundlichkeit.
>
>P.Hruschka, private communication

**Comment by ichsteffen** [comment]
**TL/DR:**
Sounds like a good idea. "Suitable" is more abstract and less restricted in use than "testable".

**Detailed Answer:**
The clearer emphasis on the aspects of "(business) suitability" and "user-friendliness" at the top level is certainly very helpful. In this way, individual stakeholders find themselves more explicitly in the top level and can more clearly sort their primary needs under it. Through the assignment of the other properties, hopefully the right "terms" can also be found in order to be able to precisely formulate the needs.

The property "suitable" is completely abstract and applicable to various objects (the software system itself, the specification, the documentation, the functional interfaces, the test cases, the runtime environment). The extension of the content of "functional suitability" (ISO25010) to "suitability" is also good in my opinion, in order to be able to address all the artifacts, processes and organizations.
But the abstractness will not make the top-down approach via "suitable" easy for some. 

The property "testable" is (presumably) primarily related to source code and executable units - and there the addressing of the property is (in my opinion) also extremely important. We all know that there are artifacts for which testability is extremely important and some for which special effort to establish testability is superfluous because the object serves a completely different purpose (e.g. PoC). Therefore, "testable" is important, but the intuitive scope may be severely limited.

The property "checkable/verifiable" goes beyond the property "testable". It is also more abstract and therefore more versatile.

**Comment by gernotstarke** [comment]
thanx Steffen - I appreciate your thoughtful contribution.

These days I will update Q42 accordingly... (as I'm currently on holiday, it might take a while, though...)

**Comment by gernotstarke** [comment]
I started changing the terms - created a branch for that. First I updated the diagrams, next the "testable" page, finally replacing all occurences within md files.

**Comment by gernotstarke** [comment]
@ichsteffen I took the liberty to extract a bit of your comment above into the home.md to explain "suitable"... And I added your name to the contributors list in "50-about-this-site.md"

**Comment by gernotstarke** [comment]
and merged...
